vector/src/main/java/im/vector/app/features/home/room/detail/TimelineViewModel.kt

```diff
@@ -626,13 +626,17 @@ class TimelineViewModel @AssistedInject constructor(
         viewModelScope.launch {
             when (action) {
                 VoiceBroadcastAction.Recording.Start -> {
+                    voiceBroadcastHelper.pausePlayback()
```

Comment:
I'm not sure if it would have been better to do this in `StartVoiceBroadcastUseCase.startVoiceBroadcast` (after checking if the user is allowed to start a VB). wdyt?

Reply:
to do that I need to inject the `VoiceBroadcastHelper` in `StartVoiceBroadcastUseCase` but `StartVoiceBroadcastUseCase` is already inject in `VoiceBroadcastHelper`